pkg/container-disk/container-disk_test.go

```diff
@@ -92,19 +92,25 @@ var _ = Describe("ContainerDisk", func() {
 				Entry("raw disk", "raw"),
 			)
 			It("by verifying error when no disk is present", func() {
+				vmi := libvmi.New()
 
-				vmi := api.NewMinimalVMI("fake-vmi")
-				appendContainerDisk(vmi, "r0")
+				By("Trying to get the volume mount directory on guest and expecting an error")
+				volumeMountDir := GetVolumeMountDirOnGuest(vmi)
+				_, err := os.Stat(filepath.Join(volumeMountDir, "disk_0.img"))
+				Expect(err).To(HaveOccurred())
+				Expect(err).To(MatchError(os.IsNotExist, "IsNotExist"))
```

Comment:
The second Expect should be sufficient. If not error occurred, it won't match.
More curiously though, this test effectively tested nothing before?!